chevronlib/keyManipulation.go

```diff
@@ -46,8 +46,9 @@ func ChangeKeyPassword(keyData, currentPassword, newPassword string) (newKeyData
 	if err != nil {
 		return
 	}
+	newKeyData, err = pgpBackend.GetPrivateKeyAsciiReencrypt(ctx, fp, currentPassword, newPassword)
 
-	newKeyData, err = pgpBackend.GetPrivateKeyAscii(ctx, fp, newPassword)
+	_ = pgpBackend.DeleteKey(ctx, fp) // Clean key after changing password
```

Comment:
Is it safe to ignore the error here?

Reply:
Yep, the error here can only be "key not found"